docs/reference/platforms/windows/index.rst

```diff
@@ -166,6 +182,51 @@ However, if you need to override this default value, you can define
 :attr:`version_triple` in your app settings. If provided, this value will be used in the
 MSI configuration file instead of the auto-generated value.
 
+Installer options
+~~~~~~~~~~~~~~~~~
+
+.. currentmodule:: install_option
+
+Windows MSI installers are able to present a panel of optional features to the user
+as part of the installation process. These features are binary flags which can then be
+used by a post-install script to perform additional installation behaviors.
+
+Installer options are defined using a TOML array of tables - each option is in
+group named ``[[ toga.briefcase.app.<app name>.install_option ]]``. Each option
+must define the following keys:
+
+.. attribute:: name
+
+An identifier for the option. This name must be a valid Python identifier; the list of
+options specified for an app must be unique when converted into upper case (i.e., you
+cannot have ``value`` and ``VALUE`` in the same configuration).
+
+.. attribute:: title
+
+A short human-readable label describing the option. Any value provided will be converted
+into a string for display to the user.
+
+.. attribute:: description
+
+A longer description of the purpose of the option. Any value provided will be converted
+into a string for display to the user.
+
+.. attribute:: default
+
+A Boolean describing the initial value of the option in the GUI. If not provided,
+defaults to ``False``.
+
+Using installer options
+-----------------------
+
+When an installer option is defined, the value of the option will be passed to the
+post-install script as a command line argument. If your installer defines an option
+with a name of ``foo``, the argument ``--foo=1`` will be passed to your post-install
+script if the option is selected by the user ``--foo=0`` will be passed if the option
```

Comment:
> It's not as difficult as you might think, especially when you know the format the arguments will be presented in.
>
> Example arg parsing

That's not too bad, but it's still a lot of boilerplate that every script will need to work out for itself, with lots of opportunities to make a mistake. Since this information is going to go into variables anyway, can we save the user that effort by setting the variables ourselves? If WiX doesn't support it, we could generate a wrapper script.